api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -56,11 +44,7 @@
   public ResponseEntity<Void> removeUserFromAuthDomain(String groupName, AuthDomainRequest request) {
     User user = userDao.findUserByEmail(request.getEmail());
     DataAccessLevel previousAccess = user.getDataAccessLevel();
-    try {
-      fireCloudService.removeUserFromGroup(request.getEmail(), groupName);
-    } catch (ApiException e) {
-      ExceptionUtils.convertFirecloudException(e);
-    }
+    fireCloudService.removeUserFromGroup(request.getEmail(), groupName);
```

Comment:
Side comment and existing issue - our prolific use of unchecked exceptions sets off some alarms for me. I suppose Swagger Spring codegen doesn't give us a choice at the top level, but something to consider for the lower level services.

Reply:
Yeah, we should have a conversation about this.

There's a question about where is the right place to throw the runtime exception that Swagger handles. Originally I thought this should happen always at the top (in the controller layer), which is why FireCloudService threw ApiException. But in most other places we've chosen to instead throw it from our lower level services... which makes the controller code much simpler, but makes it hard to tell what kinds of errors to expect from our services.

If we do decide to start throwing checked exceptions again, there's a question of whether we should be throwing exceptions like ApiException or more specific exceptions, which makes it more explicit about what error conditions we expect, but makes the client code have to handle more cases.